### ISSUE TITLE
Compilation error on file lib/dynamo/connection.ex

### ISSUE THREAD

**Issue by learod** [dynamo/dynamo#234]
Hi everyone

I don't know if that is really a bug, but I'll explain you.

When I try to compile with `mix do compile`, it's throws the following error

```
== Compilation error on file lib/dynamo/connection.ex ==
** (CompileError) lib/dynamo/connection.ex:12: function aspect/1 undefined
    (stdlib) lists.erl:1336: :lists.foreach/2
    lib/dynamo/connection.ex:10: (module)
    (stdlib) erl_eval.erl:657: :erl_eval.do_apply/6
    (elixir) src/elixir.erl:156: :elixir.erl_eval/2
```

**Comment by zukunftsalick** [comment]
same issue here

**Comment by zukunftsalick** [comment]
@learod May I know how you fixed? I'm still trying to figure it out.

**Comment by learod** [comment]
I didn't fix it. I've closed because it the same that #233

**Comment by zukunftsalick** [comment]
@learod oh ok, thanks.
